src/Controller/Command/ValidateController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\Command;
+
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use App\Exception\HttpException\NotFoundHttpException;
+use App\Entity\Command;
+
+class ValidateController
+{
+    public function __invoke(Request $request, Container $container)
+    {
+        $entityManager = $container->get('entity.manager');
+        if (null === $command = $entityManager->find(Command::class, $request->get('commandId'))) {
+            throw new NotFoundHttpException();
+        }
+        $command->setStatus(true);
```

Comment:
since you decided to use a boolean, maybe status is not the right name for this property.
maybe something like validated?